specification/app/resource-manager/Microsoft.App/preview/2023-08-01-preview/AppSpaceResources.json

```diff
@@ -0,0 +1,369 @@
+            }
+          }
+        },
+        "x-ms-examples": {
+          "Create or Update App Space": {
+            "$ref": "./examples/AppSpaceResources_CreateOrUpdate.json"
+          }
+        },
+        "x-ms-long-running-operation": true
+      },
+      "delete": {
+        "tags": [
+          "AppSpaceResources"
+        ],
+        "summary": "Delete an App Space Resource.",
+        "description": "Delete an App Space Resource.",
+        "operationId": "AppSpaceResources_Delete",
+        "parameters": [
+          {
+            "$ref": "../../../../../common-types/resource-management/v3/types.json#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v3/types.json#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "name": "appSpaceName",
+            "in": "path",
+            "description": "Name of the App Space.",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "name": "spaceResourceName",
+            "in": "path",
+            "description": "Name of the App Space Resource.",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v3/types.json#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "App Space Resource deleted successfully."
+          },
+          "204": {
+            "description": "App Space Resource does not exist."
+          },
+          "default": {
+            "description": "Common error response.",
+            "schema": {
+              "$ref": "./CommonDefinitions.json#/definitions/DefaultErrorResponse"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "Delete App Space": {
+            "$ref": "./examples/AppSpaceResources_Delete.json"
+          }
+        },
+        "x-ms-long-running-operation": true
+      }
+    }
+  },
+  "definitions": {
+    "DeploymentInformation": {
+      "description": "App Space Resource Deployment Information",
+      "type": "object",
+      "properties": {
+        "id": {
+          "description": "ARM Id of the deployment",
+          "type": "string"
+        },
+        "provisioningState": {
+          "description": "Provisioning State for the deployment",
+          "type": "string"
+        }
+      }
+    },
+    "RepositoryInformation": {
+      "description": "App Space Resource Repository Information",
+      "type": "object",
+      "properties": {
+        "url": {
+          "description": "url for the repository",
+          "type": "string"
+        },
+        "branch": {
+          "description": "branch in the repository",
+          "type": "string"
+        }
+      }
+    },
+    "Object": {
```

Comment:
This is too generic property name. Also, i am not sure if arm allows object property bag.

Reply:
I grabbed it from Microsoft.Web where it's used to define Metadata - per the spec from Portal team, ARM should allow property bags, and this was what I found when searching in api specs for metadata property bag. 
Do you have a suggestion for a different way to define a property bag object?